include/SugarObjects/templates/basic/Basic.php

```diff
@@ -113,17 +159,16 @@ public function getEmailAddressConfirmOptIn($emailField)
 
         global $sugar_config;
 
-        if (!$sugar_config['email_enable_confirm_opt_in']) {
+        if (!$sugar_config['email_enable_confirm_opt_in']) {#
             global $log;
             $log->warn('Confirm Opt In is not enabled.');
-            return true;
+            return false;
```

Comment:
Use `true` as default when `email_enable_confirm_opt_in` config value is not set or `false` as disabled, so it won't block or change any email behaviour when this option disabled by configuration.